src/utils.wlk

```diff
@@ -0,0 +1,8 @@
+object utils {
```

Comment:
No en la versión descargable actual, pero sí en la versión que vamos a
usar. Si al update site le ponés que descargue de /dev en lugar de
/stable... voilà.
No entre collections, sólo para Set, y sí va a haber que actualizar la guía
cuando salga la versión.

2016-07-03 4:14 GMT-03:00 Diego Pedro <notifications@github.com>:

> In src/utils.wlk
> <https://github.com/matifreyre/camiones/pull/3#discussion_r69386718>:
>
> > @@ -0,0 +1,8 @@
> > +object utils {
>
> Really? Pero en la versión disponible para la descarga no está, o al menos
> no lo encontré / me tiro error cuando lo quería usar. De todas formas si ya
> está, también hay que actualizar la guía de lenguajes porque justamente
> dice que no existen esas operaciones (unión, intersección y diferencia)
> entre collections
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/matifreyre/camiones/pull/3/files/c712220674ea1aaf47f6e9bf0c47ad3bc1a7b571#r69386718>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AJ-EZteIoepg7_XvqxS1Q6loNSDIvfMeks5qR2FPgaJpZM4JDnLq>
> .
>